src/layouts/EditPage/EditPage.tsx

```diff
@@ -73,12 +81,28 @@ export const EditPage = () => {
       TaskItem,
       CharacterCount,
       IsomerImage,
-      Link.configure({ openOnClick: false, protocols: ["mailto"] }),
+      Link.extend({
+        priority: 100,
```

Comment:
is the change in priority req even after the parseHTML only does anything other than `.isomer-card`?